sbi/utils/sbiutils.py

```diff
@@ -310,20 +310,16 @@ def warn_on_invalid_x(num_nans: int, num_infs: int, exclude_invalid_x: bool) ->
             )
 
 
-def warn_on_invalid_x(num_nans: int, num_infs: int, exclude_invalid_x: bool) -> None:
-    """Warn if there are NaNs or Infs. Warning text depends on `exclude_invalid_x`."""
-
-    if num_nans + num_infs > 0:
-        if exclude_invalid_x:
-            logging.warning(
-                f"Found {num_nans} NaN simulations and {num_infs} Inf simulations. "
-                "They will be excluded from training."
-            )
-        else:
-            logging.warning(
-                f"Found {num_nans} NaN simulations and {num_infs} Inf simulations. "
-                "Training might fail. Consider setting `exclude_invalid_x=True`."
-            )
+def warn_on_iid_x(num_trials):
+    """Warn if more than one x was passed."""
+    if num_trials > 1:
+        logging.warning(
+            f"A batch of {num_trials} x was passed. It will be interpreted"
+            """as a batch of independent and identically distributed data
```

Comment:
does this work? `f"` only in the first line and then triple `"""` in the next lines? Will they still be interpreted as an `f-string?`

Reply:
fixed.